src/main/groovy/com/lazan/dependency/export/MavenDependencyExport.groovy

```diff
@@ -71,25 +72,28 @@ class MavenDependencyExport extends DefaultTask {
 		ModelResolveListener resolveListener = { String groupId, String artifactId, String version, File pomFile ->
 			copyAssociatedPom(groupId, artifactId, version, pomFile)
 		}
-		ModelResolver modelResolver = new ModelResolverImpl(name, project, resolveListener)
-		for (Configuration config : prepareConfigurations()) {
-			logger.info "Exporting ${config.name}..."
-			copyJars(config)
-			copyPoms(config, modelResolver)
-			if (exportSources)
-				copyAdditionalArtifacts(config, modelResolver, SourcesArtifact)
-			if (exportJavadoc)
-				copyAdditionalArtifacts(config, modelResolver, JavadocArtifact)
-		}
-		Set<String> exportedPaths = new TreeSet()
-		project.fileTree(exportDir).visit {
-			if (!it.directory) {
-				exportedPaths << it.relativePath.pathString
+		project.getAllprojects().each {currentProject ->
```

Comment:
Hi, thanks for your PR but I won't be able to accept this in it's current state. This plugin should not apply to allprojects {}. The build author should be able to apply the plugin to the projects they want.